l10n_br_nfe/models/res_partner.py

```diff
@@ -68,7 +69,9 @@ def _prepare_import_dict(self, values, model=None):
         comodel_name="res.partner", compute="_compute_nfe40_enderDest"
     )
     nfe40_indIEDest = fields.Selection(related="ind_ie_dest")
-    nfe40_IE = fields.Char(related="inscr_est")
+    nfe40_IE = fields.Char(
+        compute="_compute_nfe_data", inverse="_inverse_nfe40_IE", store=True
```

Comment:
teria algum motivo forte para botar store=True aqui?
Na minha opinião **os campos que são puramente de mapeamento de XML de documentos fiscais nunca deveriam ter store=True**. Pois senão se generalizar isso, sempre que vai importar o gerir muitos registros no Odoo ele vai ir recalculando um montão de campos XML que vc nem tem o menor interesse ate vc exportar uma NFe que é algo relativamente raro e lento por natureza (registro por registro, lentidão do webservice...) e onde devemos jogar esses calculos na minha opinão.

Reply:
Eu apenas repeti a definição no nfe40_IE como estão nos campos nfe40_cpf e nfe40_cnpj que também estão como store=True, em termos de performace é melhor ser store=True, certo? porque se esses campos forem store=False toda vez que for ler um parceiro ele vai executar o compute... e com o store=True só vai rodar o compute quando um XML for importado, certo?

Reply:
@renatonlima store=True é mais rapido em leitura sim mas vc paga na hora de escrever e tb gera riscos de bugs. Então ele iria ler do parceiro apenas se tiver os campos no XML da tela. De uma forma geral, esses campos tem equivalente genericos (nao l10n_br_nfe) e nao precisa mostrar essa aba mais no res,partner. Eu botei eles visiveis na aba NFe automatica principalemente como demostração de tecnologia do spec_driven_model e tb para debugar inicialmente caso tiver problema. Porem logo eu acho que poderemos esconder a aba NFe automatica e mostra-la apenas para quem for gerente fiscal ou algo do tipo. So na tela da NFe mesmo que a gente poderia ter um contexto para mostrar, ou seja raramente.

Alem disso sempre que bota um store=True tem que pensar nos @api.depends é isso é fonte de bug, ressalvando:
```
There are only two hard things in Computer Science: cache invalidation and naming things.

-- Phil Karlton
```

E tem uma outra coisa que se chama:

```
premature optimization is the root of all evil
Donald Knuth
```
http://wiki.c2.com/?PrematureOptimization

```
e com o store=True só vai rodar o compute quando um XML for importado, certo?
```

-> Não, vai rodar sempre que vc alterar qualquer campo no registro res.partner!

Portanto por mim tem que tem que tirar esses store=True todos aqui no res.partner pelo menos. A gente tem que pagar o custo disso na escritura e em termo de risco de bug apenas quando a gente enxergar alguma necessidade real.

Reply:
Eu deixei o nfe40_IE como store=False, existem os campos nfe40_CPF e nfe40_CNPJ que estão com store=True mas isso é porque eles estão sendo usados no _nfe_search_keys, então mantive esse eles como store=True